src/CoreWCF.Primitives/src/CoreWCF/Dispatcher/SecurityServiceDispatcher.cs

```diff
@@ -488,7 +488,7 @@ public Task OpenAsync()
 
         public Task OpenAsync(CancellationToken token)
         {
-            throw new NotImplementedException();
+            return Task.CompletedTask;
```

Comment:
looks good.. bdw this never gets called, did you hit a situation where this was getting called ?

Reply:
Yes, Indeed it was called consistenly from the sample application and a few times tests also called it, but the behaviour was not deterministic.

Wondering if it is related to having the debugger attached.

After, trying to follow it, in the end, I decided to implement the method. But still curios about what is really happening there.